### PR TITLE
Added object detection using YOLOv5

### PR DESCRIPTION
**Pull request by aman-cc**

### CODE REVIEW

File: object_detection_yolov5/processes.py [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+    ds = yolo.datasets(dataset, file_root, ann_file, train=True)
+    dl = torch.utils.data.DataLoader(ds, shuffle=True, collate_fn=yolo.collate_wrapper, pin_memory=cuda)
+    # DataPrefetcher behaves like PyTorch's DataLoader, but it outputs CUDA tensors
+    d = yolo.DataPrefetcher(dl) if cuda else dl
+    model.to(device)
+
+    for p in model.parameters():
+        p.requires_grad_(False)
+
+    iters = 200
```
Comment (main2adwitiya) [inline object_detection_yolov5/processes.py:259]: Can you please explain this?

File: object_detection_yolov5/processes.py [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+import torch
+import yolo
+import numpy as np
+
+from torchvision import transforms
+from tqdm import tqdm
+from alectio_sdk.sdk.sql_client import create_database, add_index
+
+# COCO dataset, 80 classes
+classes = (
```
Comment (main2adwitiya) [inline object_detection_yolov5/processes.py:14]: we can load classes from a json file.

**Review by main2adwitiya — CHANGES_REQUESTED** [review]
I have suggested few changes, check the comments as well.

File: object_detection_yolov5/processes.py (hunk 1 above)
Comment (aman-cc) [reply object_detection_yolov5/processes.py:259]: - For testing, gradient compute is not required.
- iters has to be removed

File: object_detection_yolov5/processes.py (hunk 2 above)
Comment (aman-cc) [reply object_detection_yolov5/processes.py:14]: Ok

**Review by main2adwitiya — APPROVED** [review]
Looks good to me now.
